index.html

```diff
@@ -0,0 +1,256 @@
+            </div>
+        </section>
+
+         <!--work section-->
+        <section class="work-section">
+            <!--frame 44-->
+            <div class="title">
+               <h1 class="title-text">My Recent Works</h1>
+           </div>
+           <div class="frame">
+               <div class="featured-img">
+                   <div class="img-holder"></div>
+                   <div class="right-block">
+                       <div class="title-post"><h1>Multi-Post Stories</h1></div>
+                       <div class="support-text"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p></div>
+           
+                           <ul class="tags">
+                               <li class="p-tech css"><a>css</a></li>
+                               <li class="p-tech htmll"><a>html</a></li>
+                               <li class="p-tech bootzz"><a>bootstrap</a></li>
+                               <li class="p-tech rrby"><a>Ruby</a></li>
+                           </ul>
+                       <button class="madbtn">See Project</button>
+                   </div>
+               </div>
+    
+               <!--frame 43-->
+               <div class="second-frame">
+                   <div class="card-works">
+                           <div class="place-hold">
+                               <div class="r-b">
+                                   <div class="title-post"><h3 class="ghg">Profesional Art Printing Data</h3></div>
+                               <div class="supportt"><h6 class="sup-txt">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h6></div>
+                               <ul class="btn-hold">
+                                   <li class="html"><a href="#" class="txt-ht">HTML</a></li>
+                                   <li class="bootss"> <a href="#" class="b-txt"> bootstrap</a> </li>
+                                   <li class="rub"><a href="#" class="rub-txt">Ruby</a></li>
+                               </ul>
+                               </div>
+                               <div class="actz">
+                                   <button class="clicks btnn">See Project</button>
+                               </div>
+                               
+                           </div>
+                   </div>
+    
+                   <div class="card-works">
+                       <div class="place-hold">
+                           <div class="r-b">
+                               <div class="title-post"><h3 class="ghg">Profesional Art Printing Data</h3></div>
+                           <div class="supportt"><h6 class="sup-txt">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h6></div>
```

Comment:
- [ ] Kindly change this `h6` tag to a `p` tag. You are required to use a paragraph tag to describe your projects in the work section. Please fix that 👍